README.rst

```diff
@@ -68,20 +68,24 @@ Installation
 ------------
 .. readme_installation
 
-To install the latest PyGranta JobQueue release from `PyPI <https://pypi.org/project/ansys-grantami-bomanalytics/>`_,
+To install the latest PyGranta BoM Analytics release from `PyPI <https://pypi.org/project/ansys-grantami-bomanalytics/>`_,
 run this command:
 
 .. code::
 
     pip install ansys-grantami-bomanalytics
 
-To install a release compatible with a specific version of Granta MI, install the
+To install a release compatible with a specific version of Granta MI install the
```

Comment:
I think the comma is needed here. If that change was based on my comment, sorry for the extra churn.

```suggestion
To install a release compatible with a specific version of Granta MI, install the
```